sarif/sarif_file_utils.py

```diff
@@ -33,15 +35,15 @@ def combine_code_and_description(code: str, description: str) -> str:
             shorter_description = textwrap.shorten(
                 description, width=length_budget_pre_continuation, placeholder=" ..."
             )
-            if len(shorter_description) < 40:
+            if len(shorter_description) < length_budget_pre_continuation - 40:
```

Comment:
What is the magic number `40` here? Should it be `4` (i.e. the length of `" ..."`)?